pom.xml

```diff
@@ -168,4 +168,76 @@
         </repository>
     </repositories>
 
+    <profiles>
```

Comment:
I saw that some projects use org.sonatype.oss:oss-parent, which simplifies the configuration, WDYT?

Reply:
Check the URL: https://github.com/sonatype/oss-parents

Reply:
oops, pity